addons/base_import_module_apps/views/ir_module_views.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="module_view_kanban_apps_inherit">
+            <field name="name">Apps Kanban Data Modules</field>
+            <field name="model">ir.module.module</field>
+            <field name="inherit_id" ref="base.module_view_kanban"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='application']" position="after">
+                    <field name="module_type"/>
+                </xpath>
+                <xpath expr="//button[@name='button_immediate_install']" position="after">
+                <button type="object" class="btn btn-primary btn-sm" name="button_immediate_install_app" attrs="{'invisible': [('state', '!=', 'app_store')]}" groups="base.group_system" context="{'module_name':name}">Activate</button>
+                </xpath>
+                <xpath expr="//t[@t-name='kanban-menu']/a[@type='edit']" position="attributes">
+                    <attribute name="context">{'module_type': module_type, 'module_name':name}</attribute>
+                    <attribute name="type">object</attribute>
+                    <attribute name="name">more_info</attribute>
+                </xpath>
+                <xpath expr="//div[contains(@class, 'oe_module_action')]/a[@type='edit']" position="attributes">
+                    <attribute name="context">{'module_type': module_type, 'module_name':name}</attribute>
+                    <attribute name="type">object</attribute>
+                    <attribute name="name">more_info</attribute>
+                </xpath>
+            </field>
+        </record>
+        <record model="ir.ui.view" id="module_tree_apps_inherit">
+            <field name="name">Apps List Data Modules</field>
+            <field name="model">ir.module.module</field>
+            <field name="inherit_id" ref="base.module_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='category_id']" position="after">
+                    <field name="module_type" invisible="1"/>
```

Comment:
Maybe allow to show the column? Otherwise, I am afraid that I miss the reason to add this hidden field.
```suggestion
                    <field name="module_type" optional="hide"/>
```